documentation/general/decouple-vs-and-net-sdk.md

```diff
@@ -136,8 +136,11 @@ To mitigate this we will be fixing the `build-server shutdown` command to be rel
 Based on the value of the `RoslynCompilerType` property, the SDK sets property `RoslynTasksAssembly` to a full path to a [Roslyn build task DLL][roslyn-build-task],
 and the SDK targets use `$(RoslynTasksAssembly)` to load the build task.
 
-The SDK also sets `RoslynTargetsPath` to the directory path of the roslyn tasks assembly. This property is used by some targets
-but it should be avoided if possible because the tasks assembly name can change as well, not just the directory path.
+The SDK also sets the following properties:
+- `RoslynTargetsPath` to the directory path of the roslyn tasks assembly. This property is used by some targets
```

Comment:
Can we add a note that even though this is called the _targets _path it's really the _tasks_ path. It's called _targets_ because of a historical quirk but it's very possible for it to point to a directory that contains no targets.